### DISCUSSION TITLE
Mention AUR packages in README

### DISCUSSION THREAD

**Discussion by database64128** [bluenviron/mediamtx#1957] [General]
I submitted [AUR](https://aur.archlinux.org/) packages for MediaMTX and I intend to keep maintaining them. The README already mentions the OpenWRT package, so I think it would be helpful to also mention the AUR packages. What do you think?

https://aur.archlinux.org/packages/mediamtx

https://aur.archlinux.org/packages/mediamtx-git

**Comment by aler9** [comment]
Hello, thanks for creating the package, i added it to the installation methods in the README: #1969

[a comment by an automated account is not shown]
